tests/ActivityLoggerTest.php

```diff
@@ -195,6 +195,40 @@ public function it_will_use_the_logged_in_user_as_the_causer_by_default()
         $this->assertEquals($userId, $this->getLastActivity()->causer->id);
     }
 
+    /**
+     * @test
+     *
+     * @requires !Travis
```

Comment:
Is there a reason why you don't want to run these tests on TravisCi? :/

Reply:
I copied the previous test and forgot to delete this 😅

Reply:
https://github.com/spatie/laravel-activitylog/pull/605/commits/8e30bfe42e179dc308c0adb65df8d8c97ad86dfa